alpine/export_test.go

```diff
@@ -0,0 +1,9 @@
+package alpine
```

Comment:
Having global state is an anti-pattern, I'm going to cut a new PR out to take care of this.

Reply:
No. In the case of testing, `export_test.go` is a well-known technique.
https://github.com/golang/go/blob/master/src/bufio/export_test.go

You can see a lot of examples here.
https://github.com/golang/go/search?q=filename%3Aexport_test.go&unscoped_q=filename%3Aexport_test.go

*_test.go can not be referred from standard files. In short, these exported functions and variables are available only in testing.